draft-ietf-ice-rfc5245bis.xml

```diff
@@ -2157,31 +2197,31 @@ markings to its connectivity checks.
 
 <t>
 When a Binding response is received, it is correlated to its Binding
-request using the transaction ID, as defined in <xref
-target="RFC5389"/>, which then ties it to the candidate pair for which
-the Binding request was sent. This section defines additional
-procedures for processing Binding responses specific to this usage of
-STUN.
+request using the transaction ID, as defined in <xref target="RFC5389"/>,
+which then ties it to the candidate pair for which the Binding request was
+sent. This section defines additional procedures for processing Binding
+responses specific to this usage of STUN.
 </t>
 
 <section title="Failure Cases">
 
 <t>
 If the STUN transaction generates a 487 (Role Conflict) error
-response, the agent checks whether it included the ICE-CONTROLLED or
-ICE-CONTROLLING attribute in the Binding request. If the request
-contained the ICE-CONTROLLED attribute, the agent MUST switch to the
+response, the agent checks whether it included an ICE-CONTROLLED or
+ICE-CONTROLLING attribute in the associated Binding request. If the request
+contained an ICE-CONTROLLED attribute, the agent MUST switch to the
 controlling role if it has not already done so. If the request
```

Comment:
Note that there are two instances of "if it has not already done so" in the paragraph. I assume you want to remove both?